catalog/transformer/catalog-transformer-pptx/src/test/java/ddf/catalog/transformer/input/pptx/PptxInputTransformerTest.java

```diff
@@ -114,10 +114,10 @@ public void testWithTitle()
                 ss.write(os);
 
                 try (ByteArrayInputStream inStr = new ByteArrayInputStream(os.toByteArray())) {
-                    TikaInputTransformer inputTransformer = new TikaInputTransformer(null,
+                    TikaInputTransformer realTransformer = new TikaInputTransformer(null,
                             mock(MetacardType.class));
-                    inputTransformer.setUseResourceTitleAsTitle(true);
-                    PptxInputTransformer t = new PptxInputTransformer(inputTransformer);
+                    realTransformer.setUseResourceTitleAsTitle(true);
+                    PptxInputTransformer t = new PptxInputTransformer(realTransformer);
```

Comment:
✏️  Nit: Probably a good idea to change this from being a single character variable